modules/material/themes/material/default/selectidp-links.twig

```diff
@@ -15,11 +15,15 @@
 
       document.querySelector('form').appendChild(idpInput);
 
-      ga('send', 'event', 'hub', 'choice', 'IdP', id);
+      {% if analytics_tracking_id is not empty %}
+        ga('send', 'event', 'hub', 'choice', 'IdP', id);
```

Comment:
I think this should be 
      gtag('event', 'IdP', {
        'event_category': 'hub',
        'event_label': 'choice',
        'value': id
      });